cupyx/scipy/special/_logsumexp.py

```diff
@@ -61,8 +61,11 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
 
     s = cp.sum(tmp, axis=axis, keepdims=keepdims)
     if return_sign:
-        sgn = cp.sign(s)
-        s *= sgn
+        if s.dtype.char == 'c':
+            sgn = s / cp.where(s == 0, 1, cp.abs(s))
+        else:
+            sgn = cp.sign(s)
+        s = cp.abs(s)
```

Comment:
I have fixed the following changes to the SciPy 1.14 code. However, it seems that there have been further changes in SciPy 1.15. 
https://github.com/scipy/scipy/blob/92d2a8592782ee19a1161d0bf3fc2241ba78bb63/scipy/special/_logsumexp.py#L122-L128

Reply:
See https://github.com/cupy/cupy/pull/8869.

Reply:
The `cupyx.scipy.special` CIs fail in SciPy 1.15, but I want to ensure that they work in NumPy 2.x. So, I plan to fix them for SciPy 1.15 in another PR. PTAL.